UI/views/schedule.html

```diff
@@ -1,4 +1,4 @@
-<div ng-controller="scheduleController" ngcloak>
+<div ngcloak>
```

Comment:
Good catch. Actually it should not be there at all - that controller was removed.